eth/handler.go

```diff
@@ -381,7 +383,7 @@ func (pm *ProtocolManager) handleMsg(p *peer) error {
 		for !unknown && len(headers) < int(query.Amount) && bytes < softResponseLimit && len(headers) < downloader.MaxHeaderFetch {
 			// Retrieve the next header satisfying the query
 			var origin *types.Header
-			if query.Origin.Hash != (common.Hash{}) {
+			if hashMode {
```

Comment:
Our of curiosity: the reason declaring the `hashMode` in a variable is for performance reasons (as insignificant as it might be)?